src/backend/utils/storage.js

```diff
@@ -57,11 +57,18 @@ module.exports = {
       .exec();
   },
 
-  getPosts: (startDate, endDate) =>
-    // Get all posts between start and end dates in the sorted set
-    redis.zrangebyscore(POSTS, startDate.getTime(), endDate.getTime()),
+  getPosts: () =>
+    // Get all posts
+    redis.zrange(POSTS, 0, -1),
 
   getPostsCount: () => redis.zcard(POSTS),
 
-  getPost: guid => redis.hgetall(guid),
+  getPost: async guid => {
+    const post = await redis.hgetall(guid);
+
+    if (Object.keys(post).length !== 0) post.guid = guid;
```

Comment:
Instead of this, why don't we just add `guid` to the `hmset()` call above in `addPost`, so it's always available on every post object in the db.

Reply:
@humphd `guid` is already in `hmset()` ([line 36](https://github.com/Seneca-CDOT/telescope/blob/2fd026ad4409298ccd69bceb9ffd600db6fad271/src/backend/utils/storage.js#L36)), but `hgetall` returns a set without the guid, that's why I added that.